src/transformers/trainer.py

```diff
@@ -1302,10 +1310,14 @@ def train(
                 )
 
         metrics = speed_metrics("train", start_time, self.state.max_steps)
+        ort_end_train_metrics = speed_metrics("train",
```

Comment:
Looks like these will be calculated for non-ort runs too? Can we remove the ort_prefix, or add these under an if?